README.md

```diff
@@ -53,6 +53,30 @@ docker run -i -t ghcr.io/tekknolagi/scrapscript:trunk apply "1 + 2"
 docker run -i -t ghcr.io/tekknolagi/scrapscript:trunk repl
 ```
 
+### The experimental compiler:
```

Comment:
This is so awesome!! What do you think about adding some prelim benchmark results? The first thing I'm wondering is whether it's worth using over the interpreter.

Would understand if you want to hold off until you can benchmark something big or standard.

Reply:
Don't have support for variants yet but would like to add them. Shouldn't be too hard. That unlocks a lot of programs

Reply:
Then we can benchmark hull.scrap or similar